Source/Diagnostics/FlushFormats/FlushFormatSensei.cpp

```diff
@@ -55,14 +55,15 @@ FlushFormatSensei::WriteToFile (
     const amrex::Vector<ParticleDiag>& particle_diags,
     int nlev, const std::string prefix, int file_min_digits,
     bool plot_raw_fields, bool plot_raw_fields_guards,
+    const bool use_pinned_pc,
```

Comment:
Could we have use `/* use_pinned_pc */` here, as in the case of the other formats?

Reply:
That routine is already using the `ignore_unused` way of ignoring arguments. I was being consistent, using the same way as the original code. Perhaps a separate PR can clean this up and use the commented name arguments instead of `ignore_unused`.